packages/compiler/src/experimental/typekit/create-diagnosable.ts

```diff
@@ -0,0 +1,50 @@
+import { ignoreDiagnostics } from "../../core/diagnostics.js";
```

Comment:
Would be good to also convert the existing typekit that call `ignoreDiagnostic` right now

Reply:
- getPagingOperation 
- getDiscriminatoedUnion
- getHttpOperation